velox/functions/sparksql/tests/MapTest.cpp

```diff
@@ -211,5 +211,23 @@ TEST_F(MapTest, complexTypes) {
        makeArrayVector<int64_t>({{1, 2}, {4, 6}})},
       arrayMapResult1);
 }
+
+TEST_F(MapTest, resultSize) {
+  auto inputVector1 = makeNullableFlatVector<int64_t>({1, 2, 3});
+  auto inputVector2 = makeNullableFlatVector<int64_t>({4, 5, 6});
+  const RowVectorPtr& input1 = makeRowVector({inputVector1, inputVector1});
+  exec::ExprSet exprSet(
```

Comment:
This test is difficult to read and understand its intent. Consider writing a simpler test to evaluate something like if(a > 5, map(b, c), map(e, f, g, h)).

Reply:
Thanks, updated.